wsd/ClientRequestDispatcher.hpp

```diff
@@ -25,15 +25,21 @@
 class ClientRequestDispatcher final : public SimpleSocketHandler
 {
 public:
-    ClientRequestDispatcher() {}
-
     static void InitStaticFileContentCache()
     {
         StaticFileContentCache["discovery.xml"] = getDiscoveryXML();
     }
 
     typedef std::function<void(bool)> AsyncFn;
 
+    /// Uninitialize static data.
+    static void uninitialize()
+    {
+        StaticFileContentCache.clear();
```

Comment:
as an aside, it looks like StaticFileContentCache is a std::map with just one entry in it :-)

Reply:
Yes, it was provisioned for other statics as well.